bootstrap/opnsense-bootstrap.sh

```diff
@@ -108,9 +108,9 @@ echo "will automatically reboot."
 
 if [ -z "${DO_YES}" ]; then
 	echo
-	echo -n "Proceed with this action? [y/N]: "
-
-	read YN
+	echo "Proceed with this action? [y/N]: "
+	
+	read -r YN
```

Comment:
Okay, that makes sense. So that mystery is solved at least; the other stuff makes sense now.

I'm on US time & have had some weird hours today with work & fam, so pardon my fatigue. Thank you automated tooling for helping make some sense of things! @fichtner I dunno if you want to play around with this some more, but I was able to get your attention & raise my concerns. Thank you for that. :)